barcode-gen.py

```diff
@@ -47,4 +49,7 @@
         print(template.format(bar=svg_content,code=dashed_code),file=file)
 
     # Convert to png
-    subprocess.check_call(['inkscape', svg_file, '--export-png='+png_file, '--export-width=991', '--export-height=306']);
+    command = ['inkscape', svg_file, '--export-png='+png_file, '--export-width=991', '--export-height=306']
+    err = subprocess.check_call(command,stdout=open(os.devnull,'w'),stderr=subprocess.STDOUT);
```

Comment:
[`subprocess.DEVNULL`](https://docs.python.org/3/library/subprocess.html#subprocess.DEVNULL) exists and can probably be used here instead of opening devnull explicitly.